docs/domains/Domain.md

```diff
@@ -75,6 +76,7 @@ The current status of the operation of the WebLogic domain. Updated automaticall
 | `allowReplicasBelowMinDynClusterSize` | Boolean | Specifies whether the number of running cluster members is allowed to drop below the minimum dynamic cluster size configured in the WebLogic domain configuration. Otherwise, the operator will ensure that the number of running cluster members is not less than the minimum dynamic cluster setting. This setting applies to dynamic clusters only. Defaults to true. |
 | `clusterName` | string | The name of the cluster. This value must match the name of a WebLogic cluster already defined in the WebLogic domain configuration. Required. |
 | `clusterService` | [Kubernetes Resource](#kubernetes-resource) | Customization affecting Kubernetes Service generated for this WebLogic cluster. |
+| `maxConcurrentShutdown` | number | The maximum WebLogic Server instances that will shutdown in parallel for this cluster when it is being partially shutdown by lowering its replica count.A value of 0 means there is no limit. Defaults to `spec.maxClusterConcurrentShutdown` (which in turn defaults to 1). |
```

Comment:
The maximum WebLogic Server instances -> The maximum number of WebLogic Server instances
shutdown -> shut down (globally, shut down (v), shutdown (n) )
Space needed after count.
which in turn defaults to 1 -> which defaults to 1